src/Admin/DistrictAdmin.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class DistrictAdmin extends AbstractAdmin
+{
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->add('name', TextType::class, [
+                'label' => 'Nom',
+                'filter_emojis' => true,
+            ])
+            ->add('countries', TextType::class, [
+                'label' => 'Pays',
+            ])
+            ->add('geoShape', TextType::class, [
```

Comment:
it's really needed this admin? if yes, do you try create one district?
I think a `TextGeoShapeDataTransformer` is missing here, no?

Reply:
No, it's needed only to modify a district from AdherentAdmin.

Reply:
This admin is not shown in the admin dashboard and menu.

Reply:
oh I see, then the fields `countries` and `geoShape` are useless here, no? as you use only name field in AdherentAdmin.